package.json

```diff
@@ -40,21 +39,24 @@
   },
   "homepage": "https://github.com/libp2p/js-libp2p-websockets#readme",
   "dependencies": {
+    "abortable-iterator": "^2.0.0",
     "class-is": "^1.1.0",
     "debug": "^4.1.1",
     "interface-connection": "~0.3.3",
+    "it-ws": "^2.1.0",
     "mafmt": "^6.0.7",
-    "multiaddr-to-uri": "^4.0.1",
-    "pull-ws": "hugomrdias/pull-ws#fix/bundle-size"
+    "multiaddr-to-uri": "^4.0.1"
   },
   "devDependencies": {
+    "abort-controller": "^3.0.0",
     "aegir": "^18.2.1",
     "chai": "^4.2.0",
     "dirty-chai": "^2.0.1",
-    "interface-transport": "~0.3.7",
+    "interface-transport": "github:libp2p/interface-transport#master",
```

Comment:
If #master, why not released? //cc @jacobheun

Reply:
That's wrong, this is outdated. We're tackling tcp first and then will circle back to this and the other transports.